app/config.py

```diff
@@ -0,0 +1,5 @@
+import os
+from decouple import config
+MONGO_URL = config('MONGO_URL')
+DB_NAME = config('DB_NAME')
+COLLECTION_NAME = config('COLLECTION_NAME')
```

Comment:
wait wth is this? it was fine before

Reply:
oh yeah I looked it up online this is like the alternative way of using environmental variables

Reply:
doesnt work. Switch to regular `os.environ`

Reply:
okay then sure